infra/pc_scripts/start_backend.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 function start_backend {
-    docker_compose_version=$(docker-compose -v 2>&1 | sed "s/.*version \([0-9]\).\([0-9]*\).*/\1\2/");
-    if [ "$docker_compose_version" -lt "124" ]; then
+    docker_compose_major=$(docker-compose -v 2>&1 | sed "s/.*version \([0-9]\).\([0-9]*\).\([0-9]*\)*/\1/");
+    docker_compose_minor=$(docker-compose -v 2>&1 | sed "s/.*version \([0-9]\).\([0-9]*\).\([0-9]*\)*/\2/");
+    if [ "$docker_compose_major" -lt "2"] && [ "$docker_compose_minor" -lt "24"]; then
```

Comment:
```suggestion
    if [ "$docker_compose_major" -le "2"] && [ "$docker_compose_minor" -le "23"]; then
```
si on garde `lt`, `2.23` passera sous les radars (le 2 majeur n'étant pas strictement inférieur à 2).

Reply:
c'est 1.24, donc `-lt "2`" ça marche, mais si tu préfères je mets `-le "1"`

Reply:
https://github.com/pass-culture/pass-culture-main/pull/1655/commits/6204015925a58a5a09505868344a6abca0198782

Reply:
my bad, je n'ai effectivement pas vérifié que c'était 1.24 et non 2.24 en version mini.

Reply:
c'est probablement + clair avec `-le` 👍